test/exercises/2023JanTutorial/kmer.chpl

```diff
@@ -0,0 +1,48 @@
+// kmer.chpl
+/*
+   usage on puma/ocelote:
+     chpl kmer.chpl
+     ./kmer
+
+   usage on laptop with podman (or docker):
+     podman run --rm -v "$PWD":/myapp -w /myapp chapel/chapel chpl kmer.chpl
+     podman run --rm -v "$PWD":/myapp -w /myapp chapel/chapel ./kmer
+
+   For docker usage, see https://chapel-lang.org/install-docker.html
+
+   Original version of kmer counting algorithm provided by
+   Aryamaan Jain (github strikeraryu) on Chapel Discourse April 7, 2021.
+
+   kmer_large_input.txt is from
+   https://www.ncbi.nlm.nih.gov/nuccore/NC_001422.1?report=fasta
+ */
+
+use Map;
+use IO;
```

Comment:
I'd switch to `use Map, IO;` to avoid the implication that each module needs its own statement (which can get verbose in bigger programs)

Reply:
Done.